packages/create-sitecore-jss/src/templates/node-headless-ssr-proxy/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "node-headless-ssr-proxy-sample",
```

Comment:
Same issue with the indents, just run `prettier`

Reply:
Fixed